### PR TITLE
Add String representations for object types.

### PR DESCRIPTION
**Pull request by lolepezy**
* [ ] I have updated the changelog in README.md

### CODE REVIEW

File: src/main/java/net/ripe/rpki/commons/util/RepositoryObjectType.java [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
+    Manifest("mft"),
+    Roa("roa"),
+    Certificate("cer"),
+    Crl("crl"),
+    Gbr("gbr"),
+    Aspa("asa"),
+    Unknown("unknown");
+
+    @Getter
+    private final String typeName;
```
Comment (erikrozendaal) [inline src/main/java/net/ripe/rpki/commons/util/RepositoryObjectType.java:16]: `fileExtension`? Although that will mess up the `Unknown` "type". Something about the duplication with the `parse` method below doesn't feel right, but it is probably OK as it is now.

File: src/main/java/net/ripe/rpki/commons/util/RepositoryObjectType.java [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
 public enum RepositoryObjectType {
 
-    Manifest, Roa, Certificate, Crl, Gbr, Aspa, Unknown;
+    Manifest("mft"),
+    Roa("roa"),
+    Certificate("cer"),
+    Crl("crl"),
+    Gbr("gbr"),
+    Aspa("asa"),
+    Unknown("unknown");
```
[the remark this answers is not in the document]
Comment (lolepezy) [reply src/main/java/net/ripe/rpki/commons/util/RepositoryObjectType.java:13]: > Maybe we can just use the name of the enum, or the extension, instead?

What do you mean?

File: src/main/java/net/ripe/rpki/commons/util/RepositoryObjectType.java (hunk 1 above)
Comment (lolepezy) [reply src/main/java/net/ripe/rpki/commons/util/RepositoryObjectType.java]: Yeah, makes sense. Adjusted.

File: src/main/java/net/ripe/rpki/commons/util/RepositoryObjectType.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (ties) [reply src/main/java/net/ripe/rpki/commons/util/RepositoryObjectType.java:15]: If we do not use the value internally, I expected most uses to also be able to just use the .name() of the enum. But if we use it internally here as well it makes sense.

**Review by wibisono — APPROVED** [review]

**Comment by ties** [comment]
Merging this: This would have prevented a bug I had

**Pull request merged**